example/README.md

```diff
@@ -2,5 +2,12 @@
 
 All examples have their own directories, with a `main.tf` that references one or more modules. See [template_example](../template_example/example/standalone).
```

Comment:
This section should be updated with info of the example provided.

Also I think that the example files should go in it's own folder inside the examples folder. Some thing like:
```sh
example/
 ├── standalone_hive
 └── README.md
```